dataRetriever/shardedData/shardedData.go

```diff
@@ -194,21 +194,6 @@ func (sd *shardedData) MergeShardStores(sourceCacheId, destCacheId string) {
 	sd.mutShardedDataStore.Unlock()
 }
 
```

Comment:
leave the function. it will be needed when shard splits are implemented.

Reply:
@sasurobert, I removed the function according to previous review of @AdoAdoAdo.